### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by julienmartinlevrai**
Remove compiler warnings and upgrade dss

### CODE REVIEW

File: src/govActions.sol [hunk 1]
```diff
@@ -1,7 +1,10 @@
-/// govActions.sol
-
-// Copyright (C) 2018-2020 Maker Ecosystem Growth Holdings, INC.
-
+// SPDX-License-Identifier: AGPL-3.0-or-later
+//
+// govActions.sol
+//
+// Copyright (C) 2018-2021 Maker Ecosystem Growth Holdings, INC.
```
Comment (iamchrissmith) [inline src/govActions.sol:5]: i think we are supposed to remove MEGH and change all these years to Dai Foundation.

File: src/DssDeploy.t.base.sol [hunk 2]
```diff
@@ -1,3 +1,5 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
```
Comment (iamchrissmith) [inline src/DssDeploy.t.base.sol:2]: do we not copyright the tests?

File: src/DssDeploy.t.sol [hunk 3]
```diff
@@ -1,3 +1,5 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
```
Comment (iamchrissmith) [inline src/DssDeploy.t.sol:1]: do we not copyright the tests?

**Review by brianmcmichael — APPROVED** [review]

**Review by gbalabasquer — APPROVED** [review]
LGTM

File: src/govActions.sol (hunk 1 above)
Comment (gbalabasquer) [reply src/govActions.sol:5]: Yes you are actually right about this.

File: src/DssDeploy.t.base.sol (hunk 2 above)
Comment (gbalabasquer) [reply src/DssDeploy.t.base.sol:2]: Sometimes yes, sometimes no, lol.

File: src/govActions.sol (hunk 1 above)
Comment (julienmartinlevrai) [reply src/govActions.sol:5]: really ? so 2018-2022 Dai Foundation ?

**Comment by julienmartinlevrai** [comment]
@iamchrissmith I did not wait for your review. I will create another PR with your suggestions.

**Comment by iamchrissmith** [comment]
no i'm sorry @julienmartinlevrai i thought i clicked submit on the review and apparently did not do that on friday.  when i clicked it today i thought it was still open.  happy to review if you create a new one, but also not critical imo

**Comment by julienmartinlevrai** [comment]
https://github.com/makerdao/dss-deploy/pull/66

**Pull request merged**